### ISSUE TITLE
Video

### ISSUE THREAD

**Issue by manfrombar** [pyrochlore/obsidian-tracker#53]
Hello, I strongly suggest a demo video for the Tracker plugin.
Thanks a lot,

**Comment by pyrochlore** [comment]
Thanks for the suggestions.

I thought about this and planed to do it after all the main features (at least the feature I need) are done.
Maybe two or three versions later.

For now, welcome to ask questions here or directly message me on official Obsidian Discord channel.

**Comment by martincodez** [comment]
Yes a video walkthrough would be great, this is a top 3 plugin for Obsidian and it does not get enough attention — if people only knew : )
